cmd/chart-up/main.go

```diff
@@ -0,0 +1,284 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+	"k8s.io/helm/pkg/chartutil"
+)
+
+var (
+	version    string
+	gitSHA     string
+	errNoChart = errors.New("no chart provided")
+)
+
+const (
+	FilePackage = "package.yaml"
```

Comment:
It seems as though everything in here is exported however this is package main and nothing is exported from it. Can you unexport all symbols that are unnecessarily exported? (types, methods, functions)